.github/workflows/push_tests.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: |
           pip install flake8
           cd client
-          npm i "eslint" "eslint-config-airbnb" "eslint-config-prettier" "eslint-loader" "eslint-plugin-filenames" "eslint-plugin-import" "eslint-plugin-jest" "eslint-plugin-jsx-a11y" "eslint-plugin-react" "eslint-plugin-react-hooks" "eslint-plugin-prettier"
```

Comment:
using `npm install`, so we'll install `eslint` and friends following the versions and deps defined in `package-lock.json`. This prevents surprises such as linter working locally, but not in GHA.

If `npm install` taking a long time is the problem, we can always cache it!